src/components/skill/forms/skill-form.tsx

```diff
@@ -149,16 +148,24 @@ export function SkillForm({ button, initialSkills }: SkillsSheetProps) {
                   { shouldValidate: true },
                 )
               }
-              defaultValue={initialSkills?.[0]?.category ?? "SOFTSKILLS"}
+              defaultValue={initialSkills?.category ?? "SOFTSKILLS"}
             >
               <SelectTrigger className="w-full">
                 <SelectValue placeholder="Proficiency Level" />
               </SelectTrigger>
               <SelectContent>
-                <SelectItem value="SOFTSKILLS">Softskills</SelectItem>
-                <SelectItem value="TECHNICAL">Technical</SelectItem>
-                <SelectItem value="LEADERSHIP">Leadership</SelectItem>
-                <SelectItem value="COMMUNICATION">Communication</SelectItem>
+                <SelectItem value="SOFTSKILLS">
+                  <SkillBadge category={"Softskills"} />
+                </SelectItem>
+                <SelectItem value="TECHNICAL">
+                  <SkillBadge category={"Technical"} />
+                </SelectItem>
+                <SelectItem value="LEADERSHIP">
+                  <SkillBadge category={"Leadership"} />
+                </SelectItem>
+                <SelectItem value="COMMUNICATION">
+                  <SkillBadge category={"Communication"} />
+                </SelectItem>
```

Comment:
Can't you use map here? Like you have list of categories [{name, value}], then:
```typescript
categories.map(category => (
<SelectItem value={category.value}>
                  <SkillBadge category={category.name} />
                </SelectItem>
)
)
``` 
To avoid repeating yourself